GOVERNANCE.MD

```diff
@@ -1,6 +1,91 @@
+
+
+
+The CC is responsible for vetting organizations that will be supported through means such as monetary sponsorship, promotion, resource provisioning, etc. for example requirements such as the organization in question having and executing a Code of Conduct or not conflicting with Node.js Foundation’s goals and priorities.
+
+The CC and entire technical community will follow any processes as may be specified by the Board relating to the intake and license compliance review of contributions, including the Node.js Foundation IP Policy.
+
+Section 7. Elections
+
+Leadership roles in Node.js Foundation Community Committee will be peer elected representatives of the community organizations.
+
+For election of persons (CC Chairperson, Advisors, etc.) a multiple-candidate method should be used, e.g.:
+
+- ●  Condorcet: http://en.wikipedia.org/wiki/Condorcet_method or
+- ●  Single Transferable Vote: http://en.wikipedia.org/wiki/Single_transferable_vote
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one position to be filled. No election is required if there is only one candidate and no objections to the candidate's election. Nominations for organizations that should be represented on the Community Committee will take place in the GitHub repository. A representative for these organizations shall be selected within the organizations by those active in it.
+
+The CC will elect from amongst voting CC members a CC Chairperson to work on building an agenda for CC meetings and collaborate with the Individual Membership Directors the wishes of the CC to the Board for a term of one year according to the Node.js Foundation’s By-laws. The CC shall hold annual elections to select a CC Chairperson; there are no limits on the number of terms a CC Chairperson may serve.
+
+Section 8. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The CC shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
+
+The CC follows a Consensus Seeking decision making model. When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+If an agenda item cannot reach a consensus a CC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be seconded by a majority of the CC or else the discussion will continue. Simple majority wins, with the following exceptions, which will require the affirmative vote of two-thirds of the members of the CC to pass:
+
+- Adding or removing members of the CC
+- Changes to the CC Charter (which also require Board approval)
+
+Section 9. Roles
+
```

Comment:
These are a bit strange in the context of the CC since they stem from such a Tech (GitHub) point of view. For example- commit access, in addition to allowing someone to commit files- also allows them to close issues, add & create labels, edit comments... etc... which is more relevant to discussion repos.

Also, someone can be given commit access but not made part of the GitHub Org... meaning they wouldn't see any private repos. (this could be a desired)

Food for thought.